internal/observability/graph_test.go

```diff
@@ -46,7 +76,7 @@ func TestEdgeNopFallbacks(t *testing.T) {
 		RoutingDelegate: "rd",
 	}
 
-	var tagsBlocklist []string
+	var tagsBlocklist map[string]struct{}
```

Comment:
probably missed this - this goes as `*metricsTagIgnore`

Reply:
yea missed a commit when I pushed.